<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CnewDeptDlg</class>
 <widget class="QDialog" name="CnewDeptDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="m_deptNameLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>41</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Department Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_deptShortNameLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>72</y>
     <width>148</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Department Short Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_paymentAccountLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>103</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Department Payment Account Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_deptName">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_deptShortName">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_paymentAccountCombo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>441</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
